app/frontend/src/features/donations/DonationsBoxMixed.tsx

```diff
@@ -0,0 +1,458 @@
+  const onSubmit = handleSubmit((data) => {
+    initiateDonation(data);
+  });
+
+  return (
+    <>
+      <form onSubmit={onSubmit} className={classes.donationsBox}>
+        {error && <Alert severity="error">{error.message}</Alert>}
+        {success && (
+          <Alert severity="success">
+            Thank you for donating to Couchers.org! We appreciate your support!
+          </Alert>
+        )}
+        {cancelled && (
+          <Alert severity="warning">The payment was cancelled.</Alert>
+        )}
+        <Typography className={classes.marginBottom2} variant="h3">
+          {DONATIONSBOX_TITLE}
+        </Typography>
+        <Controller
+          id="recurring"
+          control={control}
+          name="recurring"
+          rules={{ required: "This field is required." }}
+          defaultValue="monthly"
+          render={({ onChange, value }) => (
+            <FormControl className={classes.formGroup}>
+              <RadioGroup
+                className={classes.donationsBoxRow}
+                aria-label="recurring"
+                name="recurring-radio"
+                onChange={(e) => onChange(e.target.value === "monthly")}
+                value={value ? "monthly" : "one-time"}
+              >
+                <FormControlLabel
+                  className={classes.buttonSecondaryRadio}
+                  value="monthly"
+                  control={<Radio />}
+                  label={DONATIONSBOX_MONTHLY}
+                />
+                <FormControlLabel
+                  className={classes.buttonSecondaryRadio}
+                  value="one-time"
+                  control={<Radio />}
+                  label={DONATIONSBOX_ONETIME}
+                />
+              </RadioGroup>
+              <FormHelperText error={!!errors?.recurring?.message}>
+                {errors?.recurring?.message ??
+                  "A monthly gift helps us plan ahead and build our operations on a stable footing."}
+              </FormHelperText>
+            </FormControl>
+          )}
+        />
+
+        <Divider className={classes.marginY2} />
+
+        <div className={classes.donationsBoxRow}>
+          <div className={classes.donationsBoxSubRow}>
+            <button
+              className={`${classes.buttonSecondary} ${
+                singleAmount === DONATIONSBOX_VALUES[0].amount
+                  ? "buttonSecondaryActive"
+                  : ""
+              }`}
+              value={DONATIONSBOX_VALUES[0].amount}
+              onClick={() => setSingleAmount(DONATIONSBOX_VALUES[0].amount)}
+              key={DONATIONSBOX_VALUES[0].amount}
+            >
+              {DONATIONSBOX_VALUES[0].currency}
+              {DONATIONSBOX_VALUES[0].amount}
+            </button>
+            <button
+              className={`${classes.buttonSecondary} ${
+                singleAmount === DONATIONSBOX_VALUES[1].amount
+                  ? "buttonSecondaryActive"
+                  : ""
+              }`}
```

Comment:
`buttonSecondaryActive` won't be the actual class name. classNames is good for this:
```suggestion
              className={classNames(classes.buttonSecondary, { [classes.buttonSecondaryActive]: watchedAmount === DONATIONSBOX_VALUES[1].amount })}
```